src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcListWithActions.scala.html

```diff
@@ -17,25 +17,40 @@
 @import uk.gov.hmrc.hmrcfrontend.views.viewmodels.listwithactions.ListWithActions
 @import uk.gov.hmrc.hmrcfrontend.views.viewmodels.listwithactions.ListWithActionsAction
 @import uk.gov.hmrc.govukfrontend.views.viewmodels.content.NonEmptyHtml
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.listwithactions.ListWithActionsItem
 
 @this()
 
 @(params: ListWithActions)
 @import params._
+
+@maybeNoActions(item: ListWithActionsItem) = @{
```

Comment:
Naming: I might prefer to see this called something like `noActionsClasses`, to be a bit more specific.